app/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx

```diff
@@ -116,6 +116,18 @@ const TableOverview = ({
                 )}
               </td>
             </tr>
+            <tr>
+              <th>Advanced Targeting</th>
+              <td data-testid="experiment-targeting-config">
+                {experiment.targetingConfig?.length ? (
+                  experiment.targetingConfig.map((t) => (
+                    <p key={t?.label}>{t?.label + "- " + t?.description}</p>
```

Comment:
You need a space at the start of the dash, and also in general, for both js and python, instead of string concatenation I'd tend to prefer format strings like so

```js
{`${t?.label} - ${t?.description}`}
```